src/main/java/com/synopsys/integration/alert/web/controller/AboutController.java

```diff
@@ -41,12 +42,14 @@
     private final AboutActions aboutActions;
     private final ResponseFactory responseFactory;
     private final ContentConverter contentConverter;
+    private final AuthorizationManager authorizationManager;
```

Comment:
This class does not appear to use AuthorizationManager

Reply:
I just removed the member variable.